### PR TITLE
For AAD B2C, support global properties.

### PR DESCRIPTION
**Pull request by chenrujun**
1. For AAD B2C, Add property mapping table for version 3.x and 4.0.
2. For AAD B2C, support global properties.
3. Fix typo.

### CODE REVIEW

**Comment by chenrujun** [comment]
Hi, @stliu , @saragluna , @backwind1233 , please help to review this PR when you have time.

File: docs/src/main/asciidoc/_migration-guide-for-4.0.adoc [hunk 1]
```diff
@@ -245,6 +245,22 @@ IMPORTANT: If you're using the new `spring-cloud-azure-starter-active-directory`
...
+| *azure.activedirectory.b2c*.client-secret              | *spring.cloud.azure.active-directory.b2c*.credential.client-secret
+| *azure.activedirectory.b2c*.authorization-clients      | *spring.cloud.azure.active-directory.b2c*.authorization-clients
+| *azure.activedirectory.b2c*.login-flow                 | *spring.cloud.azure.active-directory.b2c*.login-flow
+| *azure.activedirectory.b2c*.tenant-id                  | *spring.cloud.azure.active-directory.b2c*.profile.tenant-id
+| *azure.activedirectory.b2c*.user-flows                 | *spring.cloud.azure.active-directory.b2c*.user-flows
+| *azure.activedirectory.b2c*.user-name-attribute-nam    | *spring.cloud.azure.active-directory.b2c*.user-name-attribute-nam
+|===
+
+For properties like `credential.client-id`, `credential.client-secret`, `profile.tenant-id`, if `spring.cloud.azure.active-directory.b2c.PROPERTY` is not configured, `spring.cloud.azure.PROPERTY` will be used.
+
```
Comment (saragluna) [inline docs/src/main/asciidoc/_migration-guide-for-4.0.adoc:263]: Add `NOTE:` here?

**Review by saragluna — CHANGES_REQUESTED** [review]

**Comment by chenrujun** [comment]
@saragluna , updated.

**Review by saragluna — APPROVED** [review]
LGTM.

**Pull request merged**